Sources/Workspace/ResolverPrecomputationProvider.swift

```diff
@@ -105,7 +105,7 @@ private struct LocalPackageContainer: PackageContainer {
 
     func isToolsVersionCompatible(at version: Version) -> Bool {
         do {
-            try manifest.toolsVersion.validateToolsVersion(currentToolsVersion, packagePath: "")
+            try manifest.toolsVersion.validateToolsVersion(currentToolsVersion, packageIdentity: .plain("unknown"))
```

Comment:
Is this temporary (with the identity being passed through to the function later), or is there conceptual use for a separate `.unknown` case for identity (separate from having the identity be optional)?

Reply:
this is a special case since the error is ignored and the function returns a boolean. I would like to avid adding a formal "unknown" identity since it will be tempting to use it when in almost all cases that would be a mistake